jsonrpc/filter_manager.go

```diff
@@ -375,10 +375,13 @@ func (f *FilterManager) getLogsFromBlock(query *LogQuery, block *types.Block) ([
 		return nil, err
 	}
 
-	logs := make([]*Log, 0)
+	var (
+		logIdx int
+		logs   = make([]*Log, 0)
+	)
```

Comment:
Nitpick (use `uint64` and remove the `var` block, because we can initialize variables).
```suggestion
	logIdx := uint64(0)
	logs   := make([]*Log, 0)
```

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1533/commits/d6f055a498f815258c68974f1aeae9969bf0b1c7